changelog.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- version: 1.0.8
```

Comment:
@flash-gordon in the future, please remember to set this to `unreleased` and leave `date` empty. The `"unreleased"` version has special treatment in ossy, and it's easier to append more changelog entries w/o remembering what the next version is going to be

Reply:
 doh, it's been a while since I updated changelogs